drivers/dai/intel/alh/Kconfig.alh

```diff
@@ -0,0 +1,21 @@
+# SOF ALH configuration options
+
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config DAI_INTEL_ALH
+	bool "Intel ALH driver for Dai interface"
+	select DMA
+	help
```

Comment:
If you add this before `config`,
```
DT_COMPAT_INTEL_ALH_DAI := intel,alh-dai
```
and this inside `config`
```
default $(dt_compat_enabled,$(DT_COMPAT_INTEL_ALH_DAI))
```

This will allow the driver to be enabled whenever a devicetree node has the compatible value is enabled. This will allow CI to build this driver for cAVS 2.5

Reply:
Sure, we can do it later but for now @juimonen requested to not enabling this driver until sof can use it

Reply:
No, it needs to be enabled somewhere before we can merge it. Otherwise, it will not be included in CI and we won't know if it even builds.

Reply:
@abonislawski @juimonen I think we can proceed with a) a test case, b) enable it for cAVS2.5 SOF build, c) add a new board file using this.